utils.lua

```diff
@@ -1,12 +1,17 @@
 local Utils = {}
 BuffDuty.Utils = Utils
 
-function Utils.tableContainsValue(table, val)
-    if not table or not val then
+
+function Utils.containsName(table, name, get_name_func)
```

Comment:
I do not understand these arguments. While `table, name,` make sense, i do not see the necessity of `get_name_func`.
Also, it is not used in code of `raidinfo`: 
![image](https://user-images.githubusercontent.com/43616610/82364914-98b34700-9a18-11ea-92d5-f9b9e13fc2a4.png)

Reply:
get_name_func allows you to pass in a custom function for getting the 'name' from the table... for example: `containsName(map, name, function(x) return x.name end)` where map has the structure `map[key].name `... there is only one place it is used like though, so could just make a seperate function.

Reply:
Yes, let's do it separate, since it's only 1 place where it's used.